tests/loaders/test_http_loader.py

```diff
@@ -69,127 +69,116 @@ def __init__(self, error=None, content_type=None, body=None, code=None):
-        response_mock = ResponseMock(body='', code=200)
-        callback_mock = mock.Mock()
+        response_mock = ResponseMock(body="", code=200)
         ctx = Context(None, None, None)
-        loader.return_contents(response_mock, 'some-url', callback_mock, ctx)
-        result = callback_mock.call_args[0][0]
+        result = loader.return_contents(response_mock, "some-url", ctx)
         expect(result).to_be_instance_of(LoaderResult)
         expect(result.buffer).to_be_null()
         expect(result.successful).to_be_false()
         expect(result.error).to_equal(LoaderResult.ERROR_UPSTREAM)
 
 
-class ValidateUrlTestCase(PythonTestCase):
-
+class ValidateUrlTestCase(TestCase):
     def test_with_allowed_sources(self):
         config = Config()
         config.ALLOWED_SOURCES = [
-            's.glbimg.com',
-            re.compile(r'https://www\.google\.com/img/.*')
+            "s.glbimg.com",
+            re.compile(r"https://www\.google\.com/img/.*"),
         ]
         ctx = Context(None, config, None)
         expect(
-            loader.validate(
-                ctx,
-                'http://www.google.com/logo.jpg'
-            )
+            loader.validate(ctx, "http://www.google.com/logo.jpg")
         ).to_be_false()
         expect(
-            loader.validate(
-                ctx,
-                'http://s2.glbimg.com/logo.jpg'
-            )
+            loader.validate(ctx, "http://s2.glbimg.com/logo.jpg")
         ).to_be_false()
         expect(
             loader.validate(
                 ctx,
-                '/glob=:sfoir%20%20%3Co-pmb%20%20%20%20_%20%20%20%200%20%20g.-%3E%3Ca%20hplass='
+                "/glob=:sfoir%20%20%3Co-pmb%20%20%20%20_%20%20%20%200%20%20g.-%3E%3Ca%20hplass=",  # NOQA
             )
         ).to_be_false()
         expect(
-            loader.validate(
-                ctx,
-                'https://www.google.com/img/logo.jpg'
-            )
+            loader.validate(ctx, "https://www.google.com/img/logo.jpg")
         ).to_be_true()
         expect(
-            loader.validate(ctx, 'http://s.glbimg.com/logo.jpg')).to_be_true()
+            loader.validate(ctx, "http://s.glbimg.com/logo.jpg")
+        ).to_be_true()
 
     def test_without_allowed_sources(self):
         config = Config()
         config.ALLOWED_SOURCES = []
         ctx = Context(None, config, None)
-        is_valid = loader.validate(ctx, 'http://www.google.com/logo.jpg')
+        is_valid = loader.validate(ctx, "http://www.google.com/logo.jpg")
         expect(is_valid).to_be_true()
 
 
-class NormalizeUrlTestCase(PythonTestCase):
-
+class NormalizeUrlTestCase(TestCase):
     def test_should_normalize_url(self):
-        for url in ['http://some.url', 'some.url']:
-            expect(loader._normalize_url(url)).to_equal('http://some.url')
+        for url in ["http://some.url", "some.url"]:
+            expect(loader._normalize_url(url)).to_equal("http://some.url")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _normalize_url of a client class](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)